src/mongoose_metrics.erl

```diff
@@ -292,6 +292,33 @@ merge_stats_fun(_, V1, V2) ->
 inet_stats(Port) when is_port(Port) ->
     {ok, Stats} = inet:getstat(Port, ?INET_STATS),
     Stats;
+%% TLS dist is operated by PID, not PORT directly, so we need to find the relevant port
+inet_stats(Pid) when is_pid(Pid) ->
```

Comment:
How is it related to deps upgrade?

Reply:
This is not related but came out as I was the only one to test MIM in OTP 22 and at first there was confusion which errors come from deps upgrade and which are related to OTP 22.
This issue happens on master of MIM with OTP 22. We could have a separate ticket for this but as we already had a ticket it went there.
The ticket with description of the issue (solved by Rafal Slota) is https://erlangsolutions.atlassian.net/browse/MIM-441

Reply:
Jira ticket mentions both OTP 21 and 22 [Fix OTP21,22/cluster_commands_SUITE](https://erlangsolutions.atlassian.net/browse/MIM-441)

is it OTP 22 specific? TLS dist. was working fine for OTP 21.3, i tested it less than a month ago

Reply:
It was failing for me on master with OTP 21.3 on 26th of June (no deps upgrade changes).

```
~/Downloads/TRAVIS_master_21.3_mnesia_TLS.txt:6114: Testing big_tests.tests.cluster_commands_SUITE: *** FAILED test case 1 ***
~/Downloads/TRAVIS_master_21.3_mnesia_TLS.txt:6210: Testing big_tests.tests.cluster_commands_SUITE: *** FAILED test case 2 ***
~/Downloads/TRAVIS_master_21.3_mnesia_TLS.txt:6306: Testing big_tests.tests.cluster_commands_SUITE: *** FAILED test case 3 ***

```